packages/driver/cypress/e2e/cypress/error_utils.cy.ts

```diff
@@ -5,6 +5,7 @@ allowTsModuleStubbing()
 import $stackUtils from '@packages/driver/src/cypress/stack_utils'
 import $errUtils, { CypressError } from '@packages/driver/src/cypress/error_utils'
 import $errorMessages from '@packages/driver/src/cypress/error_messages'
+import sinon from 'sinon'
```

Comment:
```suggestion

const { sinon } = Cypress
```

`Cypress` already includes `sinon`, so we can slim down the spec file by not importing it again.